role-permissions-test.json

```diff
@@ -12,12 +12,172 @@
       }
     }
   ],
+  "view-columns": [
+    {
+      "type": "grant",
+      "action": ["view"],
+      "scope": "column"
+    }
+  ],
   "delete-tables": [
     {
       "type": "grant",
       "action": ["delete"],
       "scope": "table"
     }
+  ],
+    "create-tables": [
```

Comment:
Kannst du die json bitte noch in irgendeiner Art formatieren. Also hauptsächlich stört mich die nicht einheitliche Indentation.